<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_bt_dish_Kitchenprinter_printerSet</class>
 <widget class="QDialog" name="w_bt_dish_Kitchenprinter_printerSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>170</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打印类型</string>
      </property>
     </widget>
     <widget class="lds_ComboBox" name="comboBox_printtype">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打印机型号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>150</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打印机名称</string>
      </property>
     </widget>
     <widget class="lds_ComboBox" name="comboBox_instype">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打印机编号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_no">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>150</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>纸张宽度</string>
      </property>
     </widget>
     <widget class="lds_ComboBox" name="comboBox_paperwidth">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>150</width>
        <height>28</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_ch_areano">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>所属区域</string>
      </property>
     </widget>
     <widget class="lds_ComboBox" name="comboBox_ch_areano">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>150</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_XINYE_beep">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>蜂鸣次数</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_XINYE_beep">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>75</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_XINYE_desc">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>[1,5]</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="w_bt_dish_kitchenprinter_printerset_port" name="frame_port_select">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_ch_areano_2">
     <property name="styleSheet">
      <string notr="true">color:#ff650b;</string>
     </property>
     <property name="text">
      <string>&quot;所属区域&quot;，厨打时属于该区域的打印机才会厨打</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_uppage">
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_nextpage">
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_test">
       <property name="text">
        <string>测试</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_exit">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>w_bt_dish_kitchenprinter_printerset_port</class>
   <extends>QFrame</extends>
   <header>w_bt_dish_kitchenprinter_printerset_port.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
